Tests/NIOTests/EventLoopTest.swift

```diff
@@ -272,4 +272,13 @@ public class EventLoopTest : XCTestCase {
             tries += 1
         }
     }
+
+    public func testShutdownWhileScheduledTasksNotReady() throws {
+        let group = MultiThreadedEventLoopGroup(numThreads: 1)
+        let eventLoop = group.next()
+        _ = eventLoop.scheduleTask(in: .hours(1)) {
+
+        }
```

Comment:
Nit: maybe make it clearer this should be empty by doing `_ = eventLoop.scheduleTask(in: .hours(1)) { }`

Reply:
done